### ISSUE TITLE
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

### ISSUE THREAD

**Issue by thinkasany** [nanxiaobei/antd-img-crop#318]
<img width="2035" height="1635" alt="Image" src="https://github.com/user-attachments/assets/9b34f2e3-3ea5-4b69-8bb1-5633625470a3" />

https://github.com/ant-design/ant-design/actions/runs/18430050184/job/52516177578?pr=55271

```
 upload demo a11y › Test components/upload/demo/crop-image.tsx accessibility › accessibility › component does not have any violations

    Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

    Check the render method of `ForwardRef`.

      102 |     it(`component does not have any violations`, async () => {
      103 |       jest.useRealTimers();
    > 104 |       const { container } = render(<Component />);
          |                                   ^
      105 |
      106 |       const rules = convertRulesToAxeFormat(disabledRules || []);
      107 |

      at createFiberFromTypeAndProps (node_modules/react-dom/cjs/react-dom-client.development.js:5021:28)
      at createFiberFromElement (node_modules/react-dom/cjs/react-dom-client.development.js:5035:14)
      at createChild (node_modules/react-dom/cjs/react-dom-client.development.js:6338:26)
      at reconcileChildrenArray (node_modules/react-dom/cjs/react-dom-client.development.js:6670:25)
      at reconcileChildFibersImpl (node_modules/react-dom/cjs/react-dom-client.development.js:6993:30)
      at node_modules/react-dom/cjs/react-dom-client.development.js:7098:33
      at reconcileChildren (node_modules/react-dom/cjs/react-dom-client.development.js:9701:13)
      at updateForwardRef (node_modules/react-dom/cjs/react-dom-client.development.js:9740:7)
      at beginWork (node_modules/react-dom/cjs/react-dom-client.development.js:12117:18)
      at runWithFiberInDEV (node_modules/react-dom/cjs/react-dom-client.development.js:874:13)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom-client.development.js:17641:22)
      at workLoopSync (node_modules/react-dom/cjs/react-dom-client.development.js:17469:41)
      at renderRootSync (node_modules/react-dom/cjs/react-dom-client.development.js:17450:11)
      at performWorkOnRoot (node_modules/react-dom/cjs/react-dom-client.development.js:16583:35)
      at performWorkOnRootViaSchedulerTask (node_modules/react-dom/cjs/react-dom-client.development.js:18957:7)
      at flushActQueue (node_modules/react/cjs/react.development.js:590:34)
      at process.env.NODE_ENV.exports.act (node_modules/react/cjs/react.development.js:884:10)
      at node_modules/@testing-library/react/dist/act-compat.js:47:25
      at renderRoot (node_modules/@testing-library/react/dist/pure.js:190:26)
      at render (node_modules/@testing-library/react/dist/pure.js:292:10)
      at Object.<anonymous> (tests/shared/accessibilityTest.tsx:104:35)
```
